Reply/app/src/main/java/com/materialstudies/reply/ui/home/HomeFragment.kt

```diff
@@ -39,10 +42,19 @@ import com.materialstudies.reply.util.setOutgoingTransitions
  */
 class HomeFragment : Fragment(), EmailAdapter.EmailAdapterListener {
 
+    private val args: HomeFragmentArgs by navArgs()
+
     private lateinit var binding: FragmentHomeBinding
 
     private val emailAdapter = EmailAdapter(this)
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enterTransition = MaterialFadeThrough().apply {
```

Comment:
It does. I think it looks really good actually and decided to leave it since it also reduces complexity - not having to conditionally set/not set the `enterTransition`. Here's a gif:

<img src="https://user-images.githubusercontent.com/7459254/84426849-e5daa100-abf1-11ea-9177-aa7f639f69e1.gif" data-canonical-src="https://user-images.githubusercontent.com/7459254/84426849-e5daa100-abf1-11ea-9177-aa7f639f69e1.gif" width="200" height="400" />